spec/mailers/previews/email.html.erb

```diff
@@ -0,0 +1,141 @@
+<!--from https://github.com/rails/rails/blob/6-0-stable/railties/lib/rails/templates/rails/mailers/email.html.erb-->
+
+<!DOCTYPE html>
+<html><head>
+  <meta name="viewport" content="width=device-width" />
+  <style type="text/css">
+    html, body, iframe {
+      height: 100%;
+    }
+
+    body {
+      margin: 0;
+    }
+
+    header {
+      width: 100%;
+      padding: 10px 0 0 0;
+      margin: 0;
+      background: white;
+      font: 12px "Lucida Grande", sans-serif;
+      border-bottom: 1px solid #dedede;
+      overflow: hidden;
+    }
+
+    dl {
+      margin: 0 0 10px 0;
+      padding: 0;
+    }
+
+    dt {
+      width: 120px;
```

Comment:
Just need to revert this to the width used by the original Rails component.

```suggestion
      width: 80px;
```

Reply:
Done